### PR TITLE
docs: tips for users coming from vim and vscode

### PR DESCRIPTION
**Pull request by glennsl**
This adds two new pages in the "Getting Started" with tips for vim and vscode users. Kind of a FAQ but targeted towards specific user groups. I also think these will make it easier to contribute documentation as the format lends itself well to adding self-contained tips.

### CODE REVIEW

File: docs/docs/getting-started/vim-tips.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+[`"experimental.viml"`](https://onivim.github.io/docs/configuration/settings#experimental)
+setting.
+
+<!--
+
+### How to format on save?
+
+By running the format comamnd with `autocmd`. Help, vim people?
+
+-->
```
Comment (glennsl) [inline docs/docs/getting-started/vim-tips.md:45]: I need help with this one. I think I'm on the right track here though!

**Comment by glennsl** [comment]
Also, feel free to add any tips you already have to this PR!

**Comment by bryphe** [comment]
Thanks for adding this, @glennsl - will definitely be helpful!

File: docs/docs/getting-started/vim-tips.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "oni.layout.layoutTabPosition": "top",
+```
+
+### How do I replicate my vimrc?
+
+Using the
+[`"experimental.viml"`](https://onivim.github.io/docs/configuration/settings#experimental)
+setting you can already successfully run many VimL commands like keybindings.
+But it is experimental for good reason. Many commands do not work, and there's
+no overview of what does and does not work.
```
Comment (bryphe) [inline docs/docs/getting-started/vim-tips.md:18]: Hopefully during this milestone we can get this non-experimental and have a clear guide in terms of which `:ex` commands, viml functions, and settings work in Onivim.

File: docs/docs/getting-started/vscode-tips.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+### Tips for users coming from Visual Studio Code
```
Comment (bryphe) [inline docs/docs/getting-started/vscode-tips.md:1]: This is great!

**Pull request merged**